cmd/crio/main.go

```diff
@@ -269,6 +269,18 @@ func main() {
 			return err
 		}
 
+		// Print the current cli flags
+		flagNames := c.FlagNames()
```

Comment:
Note: The `FlagNames()` function returns currently set flags only, so we cannot have the same style output as, let's say, Kubernetes would have, with all the flags and their current values printed. A bit of a bummer, but it is what it is.